text/trace/0173-messaging-semantic-conventions.md

```diff
@@ -0,0 +1,263 @@
+serve as a basis for [stabilizing](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#stable)
+the [existing semantic conventions for messaging](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/messaging.md),
+which are currently in an [experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#experimental)
+state. The goal is to declare messaging semantic conventions stable before the
+end of 2021.
+
+## Motivation
+
+Many observability scenarios involve messaging systems, event streaming, or
+event-driven architectures. For Distributed Tracing to be useful across the
+entire scenario, having good observability for messaging or eventing operations
+is critical. To achieve this, OpenTelemetry must provide stable conventions and
+guidelines for instrumenting those operations.
+
+Bringing the existing experimental semantic conventions for messaging to a
+stable state is a crucial step for users and instrumentation authors, as it
+allows them to rely on [stability guarantees](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#not-defined-semantic-conventions-stability),
+and thus to ship and use stable instrumentation.
+
+## Roadmap
+
+1. This OTEP, consisting of scenarios and a proposed roadmap, is approved and
+   merged.
+2. [Stability guarantees](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#not-defined-semantic-conventions-stability)
+   for semantic conventions are approved and merged. This is not strictly related
+   to semantic conventions for messaging but is a prerequisite for stabilizing any
+   semantic conventions.
+3. OTEPs proposing guidance for general instrumentation problems that also
+   pertain to messaging are approved and merged. Those general instrumentation
+   problems include retries and instrumentation layers.
+4. An OTEP proposing a set of attributes and conventions covering the scenarios
+   in this document is approved and merged.
+5. Proposed specification changes are verified by prototypes for the scenarios
+   and examples below.
+6. The [specification for messaging semantic conventions for tracing](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/messaging.md)
+   are updated according to the OTEP mentioned above and are declared
+   [stable](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#stable).
+
+The steps in the roadmap don't necessarily need to happen in the given order,
+some steps can be worked on in parallel.
+
+## Terminology
+
+To leverage existing standards, the terminology used in this document is based
```

Comment:
I was confused when I read this the first time. I had the impression we will use CloudEvents specification as a way to model messaging events. I understand now that that is not the case, we are merely borrowing the terminology from CloudEvents (which is good). Perhaps delete the "To leverage existing standards" part (that's what confused me, I thought we are leveraging much more than just the terminology). The rest of the sentence is sufficient.